src/stories/Forms/Input/index.tsx

```diff
@@ -7,7 +7,7 @@ const Wrapper = styled.div<{ isHidden: boolean }>`
   flex: 1;
 `;
 
-const StyledInput = styled.input<{ hasError?: boolean }>`
+export const StyledInput = styled.input<{ hasError?: boolean }>`
   display: block;
   box-sizing: border-box;
   width: 100%;
```

Comment:
So all this styling is the same for errors in Radio and Checkbox? I see at least focus should be changes, because we only want focus on tab not on click. Other than that I think inheriting the styling is a great idea!

Reply:
No. But the styling for Radio is pretty much the same as Checkbox. So I'm thinking of importing those Checkbox into Radio, not from Input.